src/libraries/Common/tests/System/Net/SslProtocolSupport.cs

```diff
@@ -20,20 +21,40 @@ public static SslProtocols SupportedSslProtocols
         {
             get
             {
```

Comment:
If this is getting called a lot, consider changing it to a one-shot initialization:

```C#
        public static SslProtocols SupportedSslProtocols { get; } = DetermineSslProtocolSupport();

        private static SslProtocols DetermineSslProtocolSupport()
        {
            // current getter.
```

Since this will always produce the same answer.

Reply:
good point. I can look into it in next round where I plan to updated the platform detection and check how much difference it will make. In general I would like to limit initial "noise" to make it easier to step through tests in debugger.